File: basic/inline variable/javascript/example1.js
/*
Contexto: Estás desarrollando una aplicación web que muestra el precio total de productos en un carrito de compras.
Tienes una variable intermedia llamada subtotal que almacena el resultado del cálculo del precio total.
*/
const calcularPrecioTotal_ = (precioUnitario, cantidad) => {
    const subtotal = precioUnitario * cantidad;  // Refactoriza esta variable intermedia
    return subtotal;
};

const calcularPrecioTotal = (precioUnitario, cantidad) => precioUnitario * cantidad;

module.exports = {calcularPrecioTotal_, calcularPrecioTotal};

File: basic/inline variable/python/src/example_1.py
"""
Contexto: Estás desarrollando un programa que calcula el área de un triángulo.
La fórmula para calcular el área es (base * altura) / 2.
Actualmente, tienes una variable intermedia llamada area que almacena el resultado del cálculo del área.
"""

def calcular_area_triangulo_(base: float, altura: float) -> float:
    area = (base * altura) / 2  # Refactoriza esta variable intermedia
    return area

calcular_area_triangulo = lambda base, altura: (base * altura) / 2